api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -249,7 +180,7 @@ The following instructions show how to enable and configure client-side export:
     ---
 
 3. **Export the DataGrid**   
-    Implement the [onExporting](/api-reference/10%20UI%20Widgets/dxDataGrid/1%20Configuration/onExporting.md '/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting') handler and call the [exportDataGrid(options)](/api-reference/50%20Common/utils/excelExporter/exportDataGrid(options).md '/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions') method in it. In the code below, this method exports the **DataGrid** as is, but you can use [ExportDataGridProps](/api-reference/50%20Common/Object%20Structures/ExportDataGridProps '/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/') to configure export settings, including [cell customization](/api-reference/50%20Common/Object%20Structures/ExportDataGridProps/customizeCell.md '/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/#customizeCell'). To save the Excel document, call the FileSaver's **saveAs** method. The `e.cancel` parameter disables the deprecated built-in export implementation with fewer capabilities.
+    Implement the [onExporting](/Documentation/ApiReference/UI_Widgets/dxDataGrid/Configuration/#onExporting) handler and call the [exportDataGrid(options)](/Documentation/ApiReference/Common/Utils/excelExporter/#exportDataGridoptions) method in it. In the code below, this method exports the **DataGrid** as is, but you can use [ExportDataGridProps](/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/) to configure export settings, including [cell customization](/Documentation/ApiReference/Common/Object_Structures/ExportDataGridProps/#customizeCell). The **DataGrid** is exported to an Excel worksheet that is created using the ExcelJS API. To save the Excel document, call the FileSaver's **saveAs** method. The `e.cancel` parameter disables the deprecated built-in export implementation with fewer capabilities.
```

Comment:
"The `e.cancel` parameter disables the deprecated built-in export implementation with fewer capabilities."
+ It is required until we completely remove the deprecated implementation. - Do we need this clarification?

Reply:
I think we do, we show this in code and should clarify this odd-looking line. When we'll manage to remove the old export I'll also remove this line from docs.